CPS-0STL/README.md

```diff
@@ -0,0 +1,158 @@
+	4. Create metrics that are practical for wallets to implement and to present clearly to users.
+2. Shorten settlement time, as defined by stakeholder-relevant metrics, in the face of even moderate adversarial activity.
+	1. Stake-based adversaries.
+	2. Attacks utilizing adversarial resources (CPU or network).
+	3. Natural disruption of infrastructure or networks (e.g., data-center or internet outages).
+4. Fall back to Praos-like security in the face of strongly adversarial conditions.
+
+It addition to the goals above, it is advisable to avoid the following potentially costly changes.
+
+1. Avoid making major changes to the existing Ouroboros protocol parameters or logic.
+2. Do not weaken Ouroboros security or substantially enlarge its attack surface.
+3. Minimize changes that increase the resource usages of Cardano nodes or the cost of operating them.
+
+Three approaches are under active consideration or development to address the settlement and finality problem.
+
+- *Voting approaches* strengthen the weight of the preferred chain, making it extremely difficult to roll back blocks on the prefix of the chain that was made weightier by a quorum (consensus) of stake-based votes. [Ouroboros Peras](https://github.com/cardano-foundation/CIPs/pull/872) is an example of this. This approach does not mitigate CPU-resource attacks.
+- *Anti-grinding approaches* make prohibitively expensive any attacks that rely on CPU resources to weaken cryptographic guarantees of the pseudo-randomness of the slot-leadership schedule. This approach does not mitigate stake-based attacks.
+- *The protocol-parameter approach* simply lowers the active slot coefficient (currently set to 5% of the slots on `mainnet`) to a somewhat lower value, so that the block production rate and settlement are faster. This approach can only moderately shorten settlement times and does not mitigate against the aforementioned attacks.
+
+
+## Open questions
+
+- Is finality on the order of a couple of minutes feasible on Cardano? What is the theoretically fastest finality possible for a Praos-like consensus algorithm?
+- What definitions of settlement and finality are most relevant to Cardano stakeholder use cases? Is a single definition common to all use cases?
+- How best can empirical data on Cardano `mainnet` settlement and finality be collected and communicated?
+- Can a single approach to faster settlement work in the face of both stake-based and resource-based adversaries?
+- Would faster settlement negatively impact or be undercut by other planned Cardano node updates?
+- What are the trade-offs between settlement speed, throughput, performance, and security?
+- To what extent would tiered pricing for transactions improve settlement times for high-value transactions.
+- Is "rollback insurance" a viable alternative to shortening settlement times?
+
+
+## Copyright
+
+This CIP is licensed under .[Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0).
```

Comment:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```
Q: given that this content is narrative rather than code, would the Creative Commons license be the more conventional choice?

In any case it does need to be called a CPS rather than a CIP 🤓 + some punctuation fixed.